nix-script-haskell/samples/test-ghc-flags.hs

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nix-script-haskell
+#!ghcFlags -XNumericUnderscores
```

Comment:
I think you mentioned that you needed `-threaded`, right? Would it be possible to add a test for that flag (or another specific one like RTS opts?) instead of something that can already be controlled using LANGUAGE pragmas?

Reply:
Let me check if there's an easy way to check at runtime for the threaded mode. I know this can be controlled with pragmas but the code as it is written will not compile unless the `ghcFlags` are passed to the compiler.

Reply:
Done!

Reply:
perfect! Thank you so much.